cmd/ethrex/initializers.rs

```diff
@@ -385,6 +387,44 @@ async fn set_sync_block(store: &Store) {
     }
 }
 
+/// Rollback the chain to the latest block we have the state for.
```

Comment:
shouldn't this be part of the blockchain crate? It would also help us with not having to add a new dependency

Reply:
We're working on a fix that includes removing this function. It's likely we'll change this function to re-execute blocks instead of deleting them from the DB, and we won't need the trie dependency.

Reply:
[e88a692](https://github.com/lambdaclass/ethrex/pull/4806/commits/e88a692730fd7d778a841b935225ea3fecfbee98)